src/ape_ethereum/provider.py

```diff
@@ -286,15 +286,15 @@ def estimate_gas_cost(self, txn: TransactionAPI, block_id: Optional[BlockID] = N
                     tx_to_trace[key] = val
 
             try:
-                trace = self._trace_call([tx_to_trace, "latest"])
+                call_trace = self._trace_call([tx_to_trace, "latest"])
             except Exception:
-                trace = None
+                call_trace = None
 
+            traces = tuple()
```

Comment:
this should probably be `None`

```suggestion
            traces = None
```